### ISSUE TITLE
Add support for Fusion2Spjere

### ISSUE THREAD

**Issue by himynamesdave** [trek-view/gopro-frame-maker#13]
Like we allow for MAX2Sphere

```
-m custom MAX2Sphere path, default: ./MAX2sphere/MAX2sphere
```

Should allow user to pass custom path to Fusion2Sphere (-u)

If MAX2Sphere arguement or Fusion2Sphere argument passed by user, script should include that processing step.

**Comment by karamvirup** [comment]
Hi! @himynamesdave as it is fusion is there two videos (one from front & other from back)? 
If there are two videos(front/back) then will there also be an option for single video for fusion. 

As for max there's only one video.

**Comment by himynamesdave** [comment]
No. For Fusion2sphere option it is always 2 videos (front/back) or 2 jpgs (front back)

Also need to support ability to pass directory of file pairs from frame-maker (https://github.com/trek-view/fusion2sphere/issues/2)

**Comment by karamvirup** [comment]
@himynamesdave please let me know if it is working for you. will also be checking it on monday if need be.

**Comment by himynamesdave** [comment]
@karamvirup can you please complete fusion2sphere-batch, and use fusion2sphere-batch only in this script
